index.html

```diff
@@ -9,691 +9,80 @@
     dt {color: black}
     </style>
       <link href="https://www.w3.org/StyleSheets/TR/2016/W3C-ED" rel="stylesheet" type="text/css">
```

Comment:
This can be removed, because ReSpec will automatically apply styles based on the document type.

Reply:
Fixed.